Tools/ardupilotwaf/chibios.py

```diff
@@ -103,7 +103,7 @@ def wsl2_prereq_checks(self):
         except subprocess.CalledProcessError:
             #if where.exe can't find the file it returns a non-zero result which throws this exception
             where_python = ""
-        if not where_python or "\Python\Python" not in where_python or "python.exe" not in where_python:
+        if not where_python or "python.exe" not in where_python:
```

Comment:
```suggestion
        if "python.exe" not in where_python:
```

Reply:
I've applied your suggestion and it tests OK